typha/fv-tests/server_test.go

```diff
@@ -779,9 +781,10 @@ var _ = Describe("With an in-process Server with short ping timeout", func() {
 			BeforeEach(func() {
 				err := w.Encode(syncproto.Envelope{
 					Message: syncproto.MsgClientHello{
-						Hostname: "me",
-						Version:  "test",
-						Info:     "test info",
+						Hostname:                 "me",
+						Version:                  "test",
+						Info:                     "test info",
+						SupportsModernPolicyKeys: true,
```

Comment:
Should probably have a test that verifies that clients that don't set this are rejected.